apps/minifront/src/state/shared.ts

```diff
@@ -88,16 +90,31 @@ export const { assets, useAssets } = createZQuery({
   },
 });
 
+export const { gasPrices, useGasPrices } = createZQuery({
+  name: 'gasPrices',
+  fetch: getGasPrices,
+  getUseStore: () => useStore,
+  get: state => state.shared.gasPrices,
+  set: setter => {
+    const newState = setter(useStore.getState().shared.gasPrices);
+    useStore.setState(state => {
+      state.shared.gasPrices = newState;
+    });
+  },
+});
```

Comment:
thought: If this fetch is moved to `assembleRequest()` can delete

Reply:
kept this and moved `await getGasPrices()` fetch into async function in [createSendSlice](https://github.com/penumbra-zone/web/pull/1698/files#diff-b771e10f5243fc1cb84e351a8420cc7585eef71a057e4cf373871a8e59ec435aR52).